v6.2.0: run in dr-auto-sync mode, tikv will down after reset min resovled ts

## Bug Report

<!-- Thanks for your bug report! Don't worry if you can't fill out all the sections. -->

### What version of TiKV are you using?
<!-- You can run `tikv-server --version` -->

sh-4.2# ./tikv-server -V
TiKV
Release Version:   6.2.0-alpha
Edition:           Community
Git Commit Hash:   c7234fdb403fa5b69ce69ca91501448ac5194236
Git Commit Branch: heads/refs/tags/v6.2.0
UTC Build Time:    2022-07-29 15:07:02
Rust Version:      rustc 1.62.0-nightly (7c4b47696 2022-04-30)
Enable Features:   pprof-fp jemalloc mem-profiling portable sse test-engine-kv-rocksdb test-engine-raft-raft-engine cloud-aws cloud-gcp cloud-azure
Profile:           dist_release

### What operating system and CPU are you using?
<!-- If you're using Linux, you can run `cat /proc/cpuinfo` -->

### Steps to reproduce
<!-- If possible, provide a recipe for reproducing the error. A complete runnable program is good. -->
Switch from primary cluster to backup cluster when replication mode works in sync_recover mode, after that, use following command to set min resolved ts for every tikvs.

tiup ctl:v6.2.0 tikv --host xxtikvxxx reset-to-version -v xxx

Plan:  1110238
Case: oltp_tiup_dr_autosync_fun_004
clinic: 7126571720171612607

### What did you expect?
After reset version, cluster works fine.


### What did happened?
```

[2022/08/01 00:59:44.859 +08:00] [INFO] [util.go:48] ["Step: Reset to MinResolvedTs"] [callBy=drcluster.go:824]
[2022/08/01 00:59:44.859 +08:00] [INFO] [drcluster.go:827] ["will run cmd"] [cmd:="tiup ctl:v6.2.0 pd -u http://pd3-peer.endless-oltp-tps-1110238-1-959:2379 min-resolved-ts"]
[2022/08/01 00:59:45.290 +08:00] [INFO] [drcluster.go:829] ["cmd output"] ["cmd output:"="{\n  \"is_real_time\": true,\n  \"min_resolved_ts\": 434971950060666963,\n  \"persist_interval\": \"1s\"\n}"]
[2022/08/01 00:59:45.290 +08:00] [INFO] [drcluster.go:834] [minResolvedTs] [minResolvedTsInt:=434971950060666940]
[2022/08/01 00:59:45.290 +08:00] [INFO] [drcluster.go:838] ["Will run cmd"] [command="tiup ctl:v6.2.0 tikv --host tikv4-peer:20160 reset-to-version -v 434971950060666944"]
[2022/08/01 00:59:46.703 +08:00] [INFO] [drcluster.go:840] ["cmd output"] ["cmd output:"=]
[2022/08/01 00:59:46.703 +08:00] [INFO] [drcluster.go:838] ["Will run cmd"] [command="tiup ctl:v6.2.0 tikv --host tikv5-peer:20160 reset-to-version -v 434971950060666944"]
2022-08-01T00:59:45.293+0800	INFO	k8s/client.go:131	it should be noted that a long-running command will not be interrupted even the use case has ended. For more information, please refer to https://github.com/pingcap/test-infra/discussions/129
2022-08-01T00:59:46.706+0800	INFO	k8s/client.go:131	it should be noted that a long-running command will not be interrupted even the use case has ended. For more information, please refer to https://github.com/pingcap/test-infra/discussions/129
2022-08-01T00:59:47.768+0800	INFO	k8s/client.go:131	it should be noted that a long-running command will not be interrupted even the use case has ended. For more information, please refer to https://github.com/pingcap/test-infra/discussions/129
[2022/08/01 00:59:47.765 +08:00] [INFO] [drcluster.go:840] ["cmd output"] ["cmd output:"=]
[2022/08/01 00:59:47.765 +08:00] [INFO] [drcluster.go:838] ["Will run cmd"] [command="tiup ctl:v6.2.0 tikv --host tikv6-peer:20160 reset-to-version -v 434971950060666944"]
[2022/08/01 00:59:48.673 +08:00] [INFO] [drcluster.go:840] ["cmd output"] ["cmd output:"=]
[2022/08/01 00:59:48.673 +08:00] [INFO] [util.go:48] ["Step: Verify bank ACID."] [callBy=drautosync.go:276]
[2022/08/01 00:59:48.696 +08:00] [INFO] [run.go:335] ["[%s] SELECT SUM(balance) to verify use tso %d"] [client=bank2] [tso=434972075933040644]
[2022/08/01 01:29:30.699 +08:00] [ERROR] [run.go:364] ["Error 1105: context deadline exceeded"] [stack="github.com/pingcap/endless/pkg/workload/bank.(*Bank2Client).Verify\n\t/home/project/endless/pkg/workload/bank/run.go:364\ngithub.com/pingcap/endless/testcase/oltp/tiuptest/drautosync.VerifyACID\n\t/home/project/endless/testcase/oltp/tiuptest/drautosync/drautosync.go:280\ngithub.com/pingcap/endless/testcase/oltp/tiuptest/drautosync.DownPrimaryAndVerifyACID\n\t/home/project/endless/testcase/oltp/tiuptest/drautosync/drautosync.go:311\ngithub.com/pingcap/endless/testcase/oltp/tiuptest/drautosync.(*DrCaseOptions).SwitchToBackupCluster.func3\n\t/home/project/endless/testcase/oltp/tiuptest/drautosync/drautosync.go:195\ngithub.com/pingcap/endless/pkg/util.WithGinkgoRecover.func1\n\t/home/project/endless/pkg/util/fastfail.go:18\ngolang.org/x/sync/errgroup.(*Group).Go.func1\n\t/root/go/pkg/mod/golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/errgroup/errgroup.go:57"]
```
tikv keep reporting following error, pd mark tikvs disconnection.
![image](https://user-images.githubusercontent.com/9443637/182103373-6e0a0800-95d6-4e92-b7a1-b4642c92d024.png)

--- a/src/server/reset_to_version.rs
+++ b/src/server/reset_to_version.rs
@@ -134,8 +134,10 @@ impl ResetToVersionWorker {
             box_try!(wb.delete_cf(CF_WRITE, &key));
             box_try!(wb.delete_cf(CF_DEFAULT, default_key.as_encoded()));
         }
-        wb.write().unwrap();
-        wb.clear();
+        if !wb.is_empty() {
+            wb.write().unwrap();
+            wb.clear();
+        }
         Ok(has_more)
     }
 
@@ -165,7 +167,10 @@ impl ResetToVersionWorker {
                 break;
             }
         }
-        wb.write().unwrap();
+        if !wb.is_empty() {
+            wb.write().unwrap();
+            wb.clear();
+        }
         Ok(has_more)
     }
 }

